src/array_api_extra/_lib/_testing.py

```diff
@@ -173,26 +228,14 @@ def xp_assert_close(
     actual = _prepare_for_test(actual, xp)
     desired = _prepare_for_test(desired, xp)
 
-    if is_cupy_namespace(xp):
-        xp.testing.assert_allclose(
-            actual, desired, rtol=rtol, atol=atol, err_msg=err_msg
-        )
-    elif is_torch_namespace(xp):
-        xp.testing.assert_close(
-            actual, desired, rtol=rtol, atol=atol, equal_nan=True, msg=err_msg or None
-        )
-    else:
-        import numpy as np  # pylint: disable=import-outside-toplevel
-
-        # JAX/Dask arrays work directly with `np.testing`
-        assert isinstance(rtol, float)
```

Comment:
This was probably added to avoid the pyright error below? I don't think pyright should make us do this sort of thing.